lib/pbench/test/unit/config/pbench-server.cfg

```diff
@@ -4,6 +4,12 @@ install-dir = ./opt/pbench-server
 [pbench-server]
 pbench-top-dir = ./srv/pbench
 
+[elasticsearch]
+server = http://elasticsearch.perf.lab.eng.bos.redhat.com:9280
```

Comment:
The test environment should not be referencing internal instances.  Let's remove these and find another way to do this.

Reply:
I agree, I will look into this